src/collections/_documentation/server/plugins.md

```diff
@@ -44,19 +44,31 @@ All official plugins are tested against the latest version of Sentry, and compat
   content=__alert_content
 %}
 
--   [GitHub](https://github.com/getsentry/sentry-github)
--   [JIRA](https://github.com/getsentry/sentry-jira)
--   [Hipchat](https://github.com/getsentry/sentry-hipchat-ac)
--   [Slack](https://github.com/getsentry/sentry-slack)
--   [GitLab](https://github.com/getsentry/sentry-gitlab)
--   [Phabricator](https://github.com/getsentry/sentry-phabricator)
--   [Pivotal Tracker](https://github.com/getsentry/sentry-pivotal)
--   [Pushover](https://github.com/getsentry/sentry-pushover)
--   [Flowdock](https://github.com/getsentry/sentry-flowdock)
--   [Redmine](https://github.com/getsentry/sentry-redmine)
--   [BitBucket](https://github.com/getsentry/sentry-bitbucket)
--   [Trello](https://github.com/getsentry/sentry-trello)
--   [IRC](https://github.com/getsentry/sentry-irc)
+-   Amazon SQS
+-   Asana
+-   Clubhouse
+-   Heroku
+-   Pagerduty
+-   Phabricator
+-   Pivotal
+-   Pushover
+-   Segment
+-   Sessionstack
+-   Splunk
+-   Victorops
+
+All of the official plugins can be found in the [official sentry plugins repositroy](https://github.com/getsentry/sentry-plugins).
```

Comment:
```suggestion
Most of the official plugins can be found in the [official sentry plugins repositroy](https://github.com/getsentry/sentry-plugins).
```

This should be changed since we're including the yet to be moved plugins in the list above per my other comment.